packages/terra-dev-site/src/TerraDevSite.jsx

```diff
@@ -1,3 +1,4 @@
+import '@cerner/terra-polyfill';
```

Comment:
actually, include this here so that it also gets picked up by orion-dev-site: https://github.com/cerner/terra-application/blob/main/packages/terra-dev-site/src/site/_Site.jsx

Reply:
Talked offline, moving it would require overriding the linter to ensure the proper import order. The polyfill really should be explicitly imported first in every app entrypoint. We'll make changes for orion-dev-site separately.

Reply:
We discussed this and landed on leaving this and also importing the polyfill in orion-dev-site.